src/sentry/static/sentry/app/actionCreators/indicator.tsx

```diff
@@ -67,8 +88,17 @@ const prettyFormString = (val, model, fieldName) => {
     return descriptor.formatMessageValue(val, {...descriptor, initialData});
   }
 
-  if (val in PRETTY_VALUES) {
-    return PRETTY_VALUES[val];
+  if (PRETTY_VALUES.has(val)) {
+    return PRETTY_VALUES.get(val);
+  }
+
+  // boolean values not in PRETTY_VALUES because of typescript
+  if (val === true) {
+    return 'enabled';
+  }
+
+  if (val === false) {
+    return 'disabled';
   }
 
   return `${val}`;
```

Comment:
Are you not going to type `saveOnBlurUndoMessage` ?

Reply:
https://github.com/getsentry/sentry/pull/15411/commits/3e5c1b551d1acb2a2b98efd335b169749359d377#diff-935d87d0d10222eb587a1fc022bc3f00L114